content/engineering/people.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Feedback Guide
+permalink: /engineering/people/
+layout: layouts/page
+sidenav: true
+tags: engineering
+eleventyNavigation: 
+  parent: engineering_approach
+  key: Feedback Guide
+  order: 2
+  title: Feedback Guide
```

Comment:
If change only the `eleventyNavigation:` `title:` (i.e. not the one top `title` in the front matter) to "Feedback" it will match the production site. But not a huge deal if you keep it as is.